### PR TITLE
update careers and other tweaks

### PR DESCRIPTION
**Pull request by texas-kristen**
- adds card to /careers for Senior DevOps Engineer
- Removes card from /careers for TAM, posting is no longer up on Bamboo
- Updates /careers card images
- Updates and organizes redirects
- Updates webinar registration tab
- Updates names of a few images to make them more human readable
- Updates logo to use the newer version with `Font = Poppins` and `Hex = #0B6FCC`

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit e62de3fccc1c251aa1ba6c8b3d5b1d9c86e1d600

https://deploy-preview-1729--stoplightio.netlify.app

File: netlify/blog-posts/designing-apis-for-microservices.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ publishedDate: 2019-02-07T18:09:11.347Z
 author: Lukas Rosenstock
 title: Designing APIs for Microservices
 subtitle: How to Gain the Benefits of Microservice Architecture in API Design
-image: /images/ash-edmonds-636863-unsplash.jpg
+image: /images/teal-geometric-background.jpg
```
Comment (paulatulis) [inline netlify/blog-posts/designing-apis-for-microservices.md:13]: This does not show up for me, but if it does for you, I trust it.

Comment (paulatulis) [reply netlify/blog-posts/designing-apis-for-microservices.md:13]: Also, if you're changing this, you can remove line 34 (I'm unable to leave a comment there) "_Photo by [Ash Edmonds](https://unsplash.com/photos/XZ0xN0hmpi4) on [Unsplash](https://unsplash.com)_"

**Review by paulatulis — CHANGES_REQUESTED** [review]

File: netlify/blog-posts/designing-apis-for-microservices.md (hunk 1 above)
Comment (texas-kristen) [reply netlify/blog-posts/designing-apis-for-microservices.md:13]: Good catch, I renamed the file but then changed it back to the original name so I could search for all references, and then forgot to re-rename it 🤦

Comment (paulatulis) [reply netlify/blog-posts/designing-apis-for-microservices.md:13]: Oooooooh that makes sense then haha forgetting to change something back is the story of my life don't worry :) 

I still don't see the image in my preview, but perhaps it's a caching thing so will go ahead and approve. 

Screen Shot 2021-01-21 at 12 22 41 PM

**Review by paulatulis — APPROVED** [review]

**Pull request merged**